### PR TITLE
Dockerizes this nifty little app

### PR DESCRIPTION
**Pull request by prologic**

### CODE REVIEW

**Comment by prologic** [comment]
Turns this nifty little app into a Docker image.

```#!bash
$ docker images prologic/url-shortener
REPOSITORY               TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
prologic/url-shortener   latest              711ccb108a2c        10 minutes ago      35.11 MB
```

Demo: http://url.mills.io/

File: url_shortener/config.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 REDIS_PREFIX = os.getenv("REDIS_PREFIX", "short:")
 REDIS_HOST = os.getenv("REDIS_HOST", "localhost")
-REDIS_PORT = int(os.getenv("REDIS_PORT", "6379"))
```
Comment (pyr) [inline url_shortener/config.py]: why would you change this?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM prologic/python-runtime:2.7-onbuild
```
Comment (pyr) [inline Dockerfile]: aren't there more official python base images?

File: url_shortener/config.py (hunk 1 above)
Comment (prologic) [inline url_shortener/config.py]: I couldn't figure out a better way to do this. This was done quite quickly :) The long of it is that Docker 1.9+ going forward is deprecating the use of ``REDIS_PORT`` and ``REDIS_PORT_6379`` and other related env vars when linking containers (*AFAIK*) in favour of just populating the associated hosts files in the other container.

File: Dockerfile (hunk 2 above)
Comment (prologic) [inline Dockerfile]: **Not** ones that come in this small :)

**Pull request closed without merging**